From 2a231a003b8c4320b585fad92ee77adbad9fda47 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Thu, 4 Jun 2020 16:15:22 -0700
Subject: [PATCH] Fix build with boost-1.73.0

audiod/1.0.0-13-r25/git/src/state.cpp:90:75: error: '_1' was not declared in this scope
audiod/1.0.0-13-r25/git/src/state.cpp:225:76: error: '_1' was not declared in this scope
---
 src/state.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/state.cpp b/src/state.cpp
index 1feeeaf..2fc55a3 100644
--- a/src/state.cpp
+++ b/src/state.cpp
@@ -87,7 +87,7 @@ public:
     PhoneCallHandler()
     {
         IPC_SharedAudiodProperties::getInstance()->mPhoneStatus.sendChanges
-                (boost::bind(&PhoneCallHandler::phoneStatusChanged, this, _1));
+                (boost::bind(&PhoneCallHandler::phoneStatusChanged, this, boost::placeholders::_1));
     }
 
     void    phoneStatusChanged(const EPhoneStatus & phoneStatus)
@@ -222,7 +222,7 @@ void State::init()
     gAudiodProperties->mDisplayOn.sendChanges(&onDisplayOnChanged);
     gAudiodProperties->mAudiodCmd.setSetPropertyBehavior(new SetAudiodCommandBehavior());
     gAudiodProperties->mRecordingAudio.sendChanges(boost::
-                          bind(&State::setMediaServerIsRecording, &gState, _1));
+                          bind(&State::setMediaServerIsRecording, &gState, boost::placeholders::_1));
     try
     {
         gAudiodProperties->mRingerOn.set(false);
